test/e2e/node/pods.go

```diff
@@ -420,7 +419,7 @@ var _ = SIGDescribe("Pods Extended", func() {
 	})
 })
 
-var _ = SIGDescribe("Pods Extended (pod generation)", feature.PodObservedGenerationTracking, framework.WithFeatureGate(features.PodObservedGenerationTracking), func() {
+var _ = SIGDescribe("Pods Extended (pod generation)", framework.WithFeatureGate(features.PodObservedGenerationTracking), func() {
```

Comment:
should we promote this to conformance? I would expect at least one test to be a conformance test for the new API field

Reply:
(this conformance: https://github.com/kubernetes/kubernetes/tree/master/test/conformance)